Automattic/jetpack
# ci: Specify all tool versions in .github/versions.sh

#### Changes proposed in this Pull Request:

Rather than having .nvmrc, .github/php-version, and
.github/.pnpm-version. Because now we want to add a Composer version
too, and it's starting to get to be too many random version files.

Also, as mentioned, start specifying the version of Composer to use in
CI, as otherwise we may wind up with test failures whenever composer
updates their "plugin-api-version" value in composer.lock.

#### Jetpack product discussion


p1622828093282800-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Is CI happy?
* Check the various jobs to see if they're using the correct tool versions.
